end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -452,6 +470,40 @@ rtime(spec2)
 
 #' Checking its m/z
 mz(spec2)
+
+#[Dan: If one want to delve in more details, maybe would be useful to have such values in a table form, for both spectra?]
```

Comment:
nice suggestion and function @Danmarques29 - but if I'm not wrong we have already a function for that.

we will keep the idea to show the data in tabular format but will use these internal function(s).